share/config.fish

```diff
@@ -254,8 +200,22 @@ end
 # Some things should only be done for login terminals
 # This used to be in etc/config.fish - keep it here to keep the semantics
 #
-
 if status --is-login
+    # OS X-ism: Call path_helper and set its output to $PATH and $MANPATH.
+    if test -x /usr/libexec/path_helper
```

Comment:
Is there a way to do the test and run in one call without any output? If I omit the test and `path_helper` doesn't exist (e.g. on Linux) then it'll print out an unsightly error.